addons/SimpleTODO/SimpleTODO.gd

```diff
@@ -22,16 +22,38 @@ func _enter_tree():
 	
 	get_editor_interface().get_editor_viewport().add_child(todo_screen)
 	load_data()
+	print("TODO loaded")
 
 func _ready() -> void:
 	set_process_input(false)
 
+func set_window_layout(configuration: ConfigFile):
+	if configuration.has_section("SimpleTODO"):
+		var minimized_tabs = configuration.get_value("SimpleTODO", "minimized_tabs")
+		
+		if minimized_tabs.size() <= 0:
+			return
+		
+		for i in todo_screen.column_container.get_child_count():
+			var column = todo_screen.column_container.get_children()[i]
+			column.minimized = minimized_tabs[i]
+
+func get_window_layout(configuration: ConfigFile):
+	var new_minimized_tabs = []
+
+	if todo_screen:
+		for column in todo_screen.column_container.get_children():
+			new_minimized_tabs.append(column.minimized)
+	
+	configuration.set_value("SimpleTODO", "minimized_tabs", new_minimized_tabs)
+
 func _exit_tree():
 	todo_screen.queue_free()
 
 func make_visible(visible: bool) -> void:
-	todo_screen.visible = visible
-	set_process_input(visible)
+	if todo_screen:
```

Comment:
Why the added ifs? `todo_screen` always exists, unless someone will will free it.

I assume it was a workaround for the fact that saving plugin scripts will clear its variables, but it happens only during plugin development, so you can ignore the errors.

Reply:
Some part of the code earlier kept on returning errors here from what I remembered. Though I will check again.